wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/homescreen/HomeFragment.kt

```diff
@@ -299,10 +299,10 @@ class HomeFragment : Fragment() {
 		isAddOptionsShowing = show
 	}
 
-	private fun showCertificationAddFragment(dccHolder: DccHolder) {
+	private fun showCertificationAddFragment(dccHolder: DccHolder, fromCamera: Boolean) {
 		parentFragmentManager.beginTransaction()
 			.setCustomAnimations(R.anim.slide_enter, R.anim.slide_exit, R.anim.slide_pop_enter, R.anim.slide_pop_exit)
-			.replace(R.id.fragment_container, CertificateAddFragment.newInstance(dccHolder))
+			.replace(R.id.fragment_container, CertificateAddFragment.newInstance(dccHolder, fromCamera))
```

Comment:
The value for `fromCamera` is always false, so we could just set the value here to false instead of adding another parameter. Since this is the home fragment, I don't see a case where we would pass in true here